maxim/logger/livekit/realtime_session.py

```
@@ -64,7 +64,7 @@ def intercept_realtime_session_emit(self: RealtimeSession, *args, **kwargs):
             handle_google_input_transcription_completed(session_info, args[1])
     elif event == "error":
         scribe().debug(
-            f"[Internal]=====[{self.__class__.__name__}] error; args={args}, kwargs={kwargs}"
+            f"[Internal][{self.__class__.__name__}] error; args={args}, kwargs={kwargs}"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)